src/types.ts

```diff
@@ -75,35 +75,53 @@ type SectionType = 'autocomplete' | 'recommendations' | 'custom';
 
 export type SectionConfiguration = {
   type?: SectionType;
-  identifier: string;
   displayName?: string;
   numResults?: number;
   // This property will only take effect when using the component and not the hook
   displaySearchTermHighlights?: boolean;
   ref?: React.RefObject<HTMLElement>;
 };
 
-export interface AutocompleteSection extends SectionConfiguration {
+export interface AutocompleteSectionConfiguration extends SectionConfiguration {
   type?: 'autocomplete';
```

Comment:
What do you think about making type here required as well. For context
https://github.com/Constructor-io/constructorio-ui-autocomplete/pull/112/files#r1436618676

Reply:
The type should be optional to make it easier for consumers to use the library. If nothing is passed, we assume autocomplete.  If anything, we can add a default on our end